src/components/SearchGrid.vue

```diff
@@ -151,7 +151,9 @@ export default {
   },
   methods: {
     handleScalingChange() {
-      this.$redrawVueMasonry(); // Some elements end up taking less space
+      setTimeout(() => {
+        this.$redrawVueMasonry(); // Some elements end up taking less space
+      }, 1000); // One second should be sufficient to calculate new height
```

Comment:
Does it need to be a 1 second delay? Did you run into any problems with a lower timeout? I tried changing it here to 10ms and it seemed to work ok and it looks a lot more responsive

Reply:
I think it would vary based on the client computer's specifications. It would take less time on a faster client and more on a slower one (like mine). While I do agree that **1s** is overkill, values less than **100-150ms** were still running erratically on my machine. Are you confident **10ms** would suffice?

Reply:
I think 150ms should be good enough. I think humans hardly perceive anything faster than this, so it should be fine 👍

Reply:
I'll make the change as soon as possible :+1: